R/extremes-data.R

```diff
@@ -210,7 +212,27 @@ createDataRows <-
         {
           relatedValue <- relatedSet$value
         }
+        
+        if (!isEmptyOrBlank(x$relatedPrimary)) {
+          if (nrow(x$relatedPrimary) > nrow(x$points) ||
+              nrow(x$points) > nrow(x$relatedPrimary)) {
```

Comment:
Could these two if statements be simplified to nrow(x$related{Primary or Upchain}) != nrow(x$points) ?